test_runner/regress/test_pageserver_metric_collection.py

```diff
@@ -0,0 +1,338 @@
+#
+# Test for collecting metrics from pageserver and proxy.
```

Comment:
nit:  there are separate .py files for pageserver & proxy metrics, but both have this comment that says "pageserver and proxy"

Reply:
Ah, forgot this thanks.